src/containers/AccessPointDetails/index.js

```diff
@@ -221,6 +229,15 @@ const AccessPointDetails = ({
           errorFirmware={errorFirmware}
         />
       )}
+      {extraTabs.map(extraTab => {
+        return (
+          tab === extraTab.key && (
```

Comment:
remove the {} and return
` {extraTabs.map(extraTab => tab === extraTab.key && (`